src/Elcodi/Component/Sitemap/Profile/SitemapProfile.php

```diff
@@ -17,88 +17,68 @@
 
 namespace Elcodi\Component\Sitemap\Profile;
 
-use Elcodi\Component\Sitemap\Loader\Interfaces\EntityLoaderInterface;
-use Elcodi\Component\Sitemap\Profile\Interfaces\SitemapProfileInterface;
+use Elcodi\Component\Sitemap\Builder\SitemapBuilder;
 
 /**
  * Class SitemapProfile
  */
-class SitemapProfile implements SitemapProfileInterface
+class SitemapProfile
 {
     /**
-     * @var EntityLoaderInterface[]
+     * @var array
      *
-     * Entity loaders
+     * Languages
      */
-    protected $entityLoaders;
+    protected $languages;
 
     /**
-     * @var string
+     * @var SitemapBuilder[]
      *
-     * Path
+     * Array of SitemapBuilders
```

Comment:
Maybe it's just me, but I found this kind of comments exist only to "pass" the PHPDoc requirements. For me this stands at the same level of usefullness as `++$i; // increments i`. And it's not only this one: we're spreading pointless comments like this all over the place.